### ISSUE TITLE
[ERROR][bootstrap                ] Exception java.lang.Error: The operation completed successfully.

### ISSUE THREAD

**Issue by ghost** [elastic/elasticsearch#14080]
I get this strange error when trying to start ES on Windows, and am not sure what it means. The entire log is given below. Does anybody know what is the issue?

```
[2015-10-13 00:35:30,772][WARN ][bootstrap                ] jvm uses the client vm, make sure to run `java` with the server vm for best performance by adding `-server` to the command line
[2015-10-13 00:35:30,860][ERROR][bootstrap                ] Exception

java.lang.Error: The operation completed successfully.
at com.sun.jna.Native.open(Native Method)
at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:171)
at com.sun.jna.NativeLibrary.getInstance(NativeLibrary.java:398)
at com.sun.jna.Native.register(Native.java:1396)
at com.sun.jna.Native.register(Native.java:1156)
at org.elasticsearch.bootstrap.JNAKernel32Library.<init>(JNAKernel32Library.java:53)
at org.elasticsearch.bootstrap.JNAKernel32Library.<init>(JNAKernel32Library.java:37)
at org.elasticsearch.bootstrap.JNAKernel32Library$Holder.<clinit>(JNAKernel32Library.java:47)
at org.elasticsearch.bootstrap.JNAKernel32Library.getInstance(JNAKernel32Library.java:64)
at org.elasticsearch.bootstrap.JNANatives.addConsoleCtrlHandler(JNANatives.java:109)
at org.elasticsearch.bootstrap.Natives.addConsoleCtrlHandler(Natives.java:67)
at org.elasticsearch.bootstrap.Bootstrap.initializeNatives(Bootstrap.java:101)
at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:66)
at org.elasticsearch.bootstrap.Bootstrap.main(Bootstrap.java:245)
at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:32)
```

**Comment by clintongormley** [comment]
Looks like it may be related to this bug in JNA 4.1.0: https://github.com/java-native-access/jna/issues/517

Do you have Norton security installed by any chance?

Apparently this bug is fixed in 4.2.0.  Any reason for us not to upgrade?

**Comment by rmuir** [comment]
> Apparently this bug is fixed in 4.2.0. Any reason for us not to upgrade JNA?

There is some degree of risk i suppose, but I tested it a few weeks ago and did not encounter any issues (on linux). However, the fact that they don't even know how the fixed the bug in 4.2.0 is not encouraging.

**Comment by djurikom84** [comment]
Yes, I have Norton Security Suite. Is there something I can do now?
If it is of any help, I encountered this issue on ES 1.5.0 and 1.7.2. Once I moved to 1.4.5 it worked OK.
